### PR TITLE
Show basic preview for recycle bin items

### LINKED ISSUE #7196: Video previews in recycle bin broken.
### Description

So whenever I clicked on a video it starts out normal for I would guess 1/4 to 1/2 a second and then it turns into this covering the whole video, I also noticed for the 1/2 second it is normal the play button is white as if I am able to click it but after it bugs out it turns this gray and I cant play it neither can I change the time line. I have restarted and tested this 3 times and found out that it is re-creatable and it only happens in the recycle bin. So videos in say videos folder didn't get this bug.

### Steps To Reproduce

1. Recycle a random video
2. go to recycle bin
3. click on the video and enable the preview pane
4. see this bug happens (image linked below.)

### Expected behavior

To not extend up that high.

### Files Version

v2.0.41.0

### Windows Version

Windows 11 21H2 22000.348

### Relevant Assets/Logs

[debug.log](https://github.com/files-community/Files/files/7644026/debug.log)
[debug_fulltrust.log](https://github.com/files-community/Files/files/7644027/debug_fulltrust.log)
image

### PR DESCRIPTION
**Pull request by winston-de**
**Resolved / Related Issues**
Items resolved / related issues by this PR.
- Closes #7196

**Details of Changes**
Add details of changes here.
- Only show the basic preview for recycle bin items
- allow opening folders with double click when multiselect is enabled

**Validation**
How did you test these changes?
- [x] Built and ran the app
- [ ] Tested the changes for accessibility

I can't test that the previews work properly in the recycle bin because the full trust process is still broken, so if someone could test that it would be appreciated.

**Screenshots (optional)**
Add screenshots here.

### CODE REVIEW

File: Files/ViewModels/PreviewPaneViewModel.cs [hunk 1]
```diff
@@ -234,18 +242,8 @@ public async void UpdateSelectedItemPreview(bool downloadItem = false)
...
-                            await basicModel.LoadAsync();
-                            PreviewPaneContent = new BasicPreview(basicModel);
-                            PreviewPaneState = PreviewPaneStates.PreviewAndDetailsAvailable;
-                            return;
-                        }
-                        catch (Exception ex)
-                        {
-                            Debug.WriteLine(ex);
-                        }
+                        await LoadBasicPreviewAsync();
```
Comment (gave92) [inline Files/ViewModels/PreviewPaneViewModel.cs:245]: Perhaps we can revert this change: if there's an exception loading the basic preview we should execute `PreviewPaneContent = null;` and not return, right?

File: Files/Helpers/NavigationHelpers.cs [hunk 2]
```diff
@@ -123,7 +131,7 @@ public static async void OpenItemsWithExecutable(IShellPage associatedInstance,
         /// <param name="openSilent">Determines whether history of opened item is saved (... to Recent Items/Windows Timeline/opening in background)</param>
         /// <param name="openViaApplicationPicker">Determines whether open file using application picker</param>
         /// <param name="selectItems">List of filenames that are selected upon navigation</param>
```
Comment (d2dyno1) [inline Files/Helpers/NavigationHelpers.cs:134]: Could you add docs for `forceOpenInNewTab`. And also while at it, `args` too? @gave92

**Review by gave92 — COMMENTED** [review]
@winston-de could you check the changes I've made?
- Fixed preview not updated for folders in recycle bin
- Open folders in new tab when double clicking with multiple selected folders

**Comment by winston-de** [comment]
> @winston-de could you check the changes I've made?
> 
> * Fixed preview not updated for folders in recycle bin
> * Open folders in new tab when double clicking with multiple selected folders

Looks good, thanks

**Review by gave92 — APPROVED** [review]

**Pull request merged**